catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/SearchFormsApplication.java

```diff
@@ -292,31 +292,28 @@ private String safeGetBody(Request req) {
       return ImmutableMap.of(RESP_MSG, "Could not create, no valid template specified");
     }
 
-    // Going to check to see if the metacard exists or not
-    Metacard oldMetacard = null;
+    // The UI should not send an ID during a PUT unless the metacard already exists
```

Comment:
is it risky to make this assumption about frontend behavior?

Reply:
This was the agreed upon contract. I think due to some UI details we couldn't distinguish between a `PUT` and a `POST` since template views were adapted from the advanced query builder, so we went this route. If a bogus ID is sent, we will know and adjust accordingly, since we are verifying any sent ID against the catalog before actually creating or updating.